index.bs

```diff
@@ -3517,7 +3517,7 @@ This is because the first 37 bytes of the signed data in a FIDO U2F authenticati
 [=authenticator data=] structure, the <code>[=rpIdHash=]</code> is the FIDO U2F [=application parameter=], all
 <code>[=flags=]</code> except <code>[=UP=]</code> are always zero, and the <code>[=attestedCredentialData=]</code> and
 <code>[=authDataExtensions|extensions=]</code> are never present. FIDO U2F authentication signatures can therefore be verified by
-the same procedure as other [=assertion signatures=] generated by the [=authenticatorMakeCredential=] operation.
+the same procedure as other [=assertion signatures=] generated by the [=authenticatorGetAssertion=] operation.
```

Comment:
Or perhaps this should even be `{{PublicKeyCredential/[[DiscoverFromExternalSource]]()}}` or `navigator.credentials.get()` instead?

Reply:
I would leave it as `[=authenticatorGetAssertion=]`